src/Renderer/OBB.js

```diff
@@ -114,34 +117,40 @@ class OBB extends THREE.Object3D {
      * @return     {TileGeometry}  { description_of_the_return_value }
      */
     static extentToOBB(extent, minHeight = 0, maxHeight = 0, obb) {
```

Comment:
Do you know why this method is static ? It could be normal with `obb` being `this`, WDYT ?

Reply:
Oh I think we can't because of here ?  https://github.com/iTowns/itowns/pull/1046/commits/47370349e9d11e488cc705f67acefe84e6434911#diff-2e3fb169a574bac9596d8c12cfaa8de0R132

Reply:
i remove static function and rename `extentToOBB` to `setFromExtent`.